tests/TestCase.php

```diff
@@ -23,16 +26,8 @@ protected function getPackageProviders($app)
 
     public function getEnvironmentSetUp($app)
     {
-        $app['config']->set('database.default', 'sqlite');
-        $app['config']->set('database.connections.sqlite', [
-            'driver' => 'sqlite',
-            'database' => ':memory:',
-            'prefix' => '',
+        $app['config']->set('media.models', [
+            'user' => User::class,
         ]);
-
-        /*
-        include_once __DIR__.'/../database/migrations/create_media_table.php.stub';
-        (new \CreatePackageTable())->up();
-        */
     }
 }
```

Comment:
This can't use the user model through a separate config file. It must use the tipoff config file so we just have one source to update it.

Reply:
Thank you for the heads up. I pushed new commits.